mailbox/api/src/main/java/org/apache/james/mailbox/exception/MailboxNameException.java

```diff
@@ -17,11 +17,19 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.jmap.exceptions;
+package org.apache.james.mailbox.exception;
 
-public class MailboxNameException extends RuntimeException {
+public class MailboxNameException extends MailboxException {
```

Comment:
Maybe it would better to create an dedicated exception for your issue, such as TooLongMailboxNameException